sink_test.go

```diff
@@ -118,3 +120,77 @@ func TestFanoutSink_Sample(t *testing.T) {
 		t.Fatalf("val not equal")
 	}
 }
+
+func TestNewMetricSinkFromURL(t *testing.T) {
```

Comment:
I was starting to fan out the per-sink factory tests but they started to feel boiler-platey. I was on the fence though so with this nudge I'll drop 'em in! <3